data/live-build-config/includes.chroot/usr/lib/live/boot/9991-vyos.sh

```
@@ -0,0 +1,303 @@
+			fi
+		done
+	done
+
+	# VyOS: flat backing directory, without the per-device sub-directory that
+	# upstream uses. VyOS boots a single persistence partition and addresses
+	# every image below it as boot/<version>/rw; the persistence root must
+	# therefore be at a fixed, device-independent path.
+	backing="/run/live/persistence"
+
+	mkdir -p "${backing}"
+	old_backing="$(where_is_mounted ${device})"
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)